media/css/pebbles/components/_survey-message.scss

```diff
@@ -0,0 +1,77 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#survey-message {
+    @include border-box;
+    background: #fff0b3;
+    bottom: 0;
+    min-height: 30px;
+    overflow: hidden;
+    padding: 10px 20px;
+    position: fixed;
+    text-align: center;
+    transform: translateY(100%);
```

Comment:
Mixin here as well. 

As an aside, at this point all the browsers have supported transform unprefixed since 2014, and transition since 2013. I wonder if it's time to start phasing out some of those prefix mixins. Perhaps add PostCSS autoprefixer that would be easier to maintain in one place. Something to bikeshed in Toronto maybe.